ddtrace/internal/packages.py

```diff
@@ -10,6 +10,12 @@
 from ddtrace.internal.utils.cache import callonce
 
 
+try:
+    import importlib.metadata as importlib_metadata
+except ImportError:
+    import importlib_metadata  # type: ignore[no-redef]
```

Comment:
Why are we moving this out to the module scope instead of keeping it within the function scope (for this specific function)? I see it's still imported at the function level in some others

Reply:
it was in a local scope, I moved it to the module scope to make it available in the two other files that need to access the current Flask version.